xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm

```diff
@@ -179,26 +179,28 @@
   #end
   #template("hierarchy.vm")
   <div class="row">
-    #if ($displayContentMenu)
-      <div class="col-xs-12 col-md-5 pull-right">
-        #template("menus_content.vm")
-      </div>
-    #end
-    #if (($editor == 'wiki' || $editor == 'wysiwyg') && !$request.section)
-      <div id="editMeta" class="col-xs-12#if ($displayContentMenu) col-md-7#end">
-        #template('editmeta.vm')
-      </div>
-    #else
-      <div id="document-title" class="col-xs-12#if ($displayContentMenu) col-md-7#end"><h1>
-      #if($editor == 'rights')
-        $services.localization.render('core.editors.rights.title', [$escapetool.xml($doc.plainTitle), $doc.getURL()])
-      #elseif($editor == 'object')
-        $services.localization.render('core.editors.object.title', [$escapetool.xml($doc.plainTitle), $doc.getURL()])
-      #elseif($editor == 'class')
-        $services.localization.render('core.editors.class.title', [$escapetool.xml($doc.fullName), $doc.getURL()])
+    <dl>
+      #if ($displayContentMenu)
+        <div class="col-xs-12 col-md-5 pull-right">
+          #template("menus_content.vm")
+        </div>
       #end
-      </h1></div>
-    #end
+      #if (($editor == 'wiki' || $editor == 'wysiwyg') && !$request.section)
+        <div id="editMeta" class="col-xs-12#if ($displayContentMenu) col-md-7#end">
+          #template('editmeta.vm')
```

Comment:
Same as above, this produces invalid HTML most probably.

Reply:
That's not what I see. I see this:

```
<dl>
  <div id="editMeta" class="col-xs-12 col-md-7">
    <div id="parentinput" class="form-group">
      <label for="xwikidocparentinput">...</label>
      <input type="text" id="xwikidocparentinput" name="parent"  ..."/>
    </div>
    ...
```

which is definitely not a valid definition list for me.

Reply:
Hm, so I have to use `dl` in `edit.vm` only.